agent/gogenerate/awssdk.go

```diff
@@ -39,7 +41,7 @@ var typesOnlyTplAPI = template.Must(template.New("api").Parse(`
 {{ end }}
 `))
 
-const copyrightHeader = `// Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+const copyrightHeader = `// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
minor: Can you modify this to get the `year` string in code so that we don't have to update this every year?